resolwe_bio/tools/append_sample_info.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+"""Extract reference (FASTA) and sample names from the VCF file."""
+
+import os
+import argparse
+
+from pysam import VariantFile
+from resolwe_runtime_utils import error, warning
+
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('vcf_file', help="VCF file (can be compressed using gzip/bgzip).")
+parser.add_argument('summary', help="Summary file to append to.")
+args = parser.parse_args()
+
+try:
+    vcf = VariantFile(args.vcf_file)
+except (OSError, ValueError):
+    print(error('Input VCF file does not exist or could not be correctly opened.'))
```

Comment:
Is `print` necessary here?

Reply:
Yes, we need the JSON formatted by `error()` function printed to stdout